src/OgRoleInterface.php

```diff
@@ -5,11 +5,12 @@
  * Contains Drupal\og\OgRoleInterface.
  */
 namespace Drupal\og;
+use Drupal\user\RoleInterface;
 
 /**
  * Provides an interface defining an OG user role entity.
  */
-interface OgRoleInterface {
+interface OgRoleInterface extends RoleInterface {
```

Comment:
seems the previous PHP version don't like this:

`PHPUnit_Framework_Exception: Fatal error: Cannot inherit previously-inherited or override constant ANONYMOUS_ID from interface Drupal\og\OgRoleInterface in /home/travis/build/amitaibu/og/src/Entity/OgRole.php on line 34` (from [here](https://travis-ci.org/amitaibu/og/jobs/131386627#L531))


But not really sure why. @pfrenssen can you spot it?